tools/chocolateyInstall.ps1

```diff
@@ -74,7 +69,7 @@ if ($service) {
 
 Write-Host "Installing service: $serviceName"
 # Install the service
-& $wrapperExe install $serviceName $(Join-Path $toolsPath "consul.exe") "agent -ui-dir=$serviceUiDirectory -config-dir=$serviceConfigDirectory -data-dir=$serviceDataDirectory $packageParameters" | Out-Null
+& $wrapperExe install $serviceName $(Join-Path $toolsPath "consul.exe") "agent -ui -config-dir=$serviceConfigDirectory -data-dir=$serviceDataDirectory $packageParameters" | Out-Null
```

Comment:
We can also remove these lines: 

https://github.com/calvn/chocolatey-consul/blob/master/tools/chocolateyInstall.ps1#L9
https://github.com/calvn/chocolatey-consul/blob/master/tools/chocolateyInstall.ps1#L25
https://github.com/calvn/chocolatey-consul/blob/master/tools/chocolateyInstall.ps1#L37